verification/src/transaction_verifier.rs

```diff
@@ -211,26 +211,31 @@ impl<'a> CapacityVerifier<'a> {
     }
 
     pub fn verify(&self) -> Result<(), TransactionError> {
-        let inputs_total = self
-            .resolved_transaction
-            .input_cells
-            .iter()
-            .try_fold(Capacity::zero(), |acc, cell_meta| {
-                acc.safe_add(cell_meta.capacity())
-            })?;
-
-        let outputs_total = self
-            .resolved_transaction
-            .transaction
-            .outputs()
-            .iter()
-            .try_fold(Capacity::zero(), |acc, output| {
-                acc.safe_add(output.capacity)
-            })?;
-
-        if inputs_total < outputs_total {
-            return Err(TransactionError::OutputsSumOverflow);
+        // skip OutputsSumOverflow verification for resolved cellbase
+        // cellbase's outputs are verified by TransactionsVerifier#InvalidReward
```

Comment:
I think it's better to use `Transaction::is_cellbase` here.

Reply:
Resolved, please review again.